apps/tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,25 @@
+
+#include "CANBuffer.h"
+#include "CANBus.h"
+#include "CANMsg.h"
+#include "TutorialServo.h"
+#include "hw_bridge.h"
+#include "mbed.h"
+
+TutorialServo servo1 = TutorialServo(PA_1);
+
+CANBus can(CAN1_RX, CAN1_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
+CANMsg servoMsg;
+
+int main() {
+  while (true) {
+    if (can.read(servoMsg)) {
+      float degrees;
+      const float PERCENT_TO_DEG = 180;
```

Comment:
Use the getServoRangeInDegrees() function from your TutorialServo library to determine this value